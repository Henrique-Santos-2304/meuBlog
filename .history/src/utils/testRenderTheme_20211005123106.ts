import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(<ThemeProvider>{children}</ThemeProvider> );
};
