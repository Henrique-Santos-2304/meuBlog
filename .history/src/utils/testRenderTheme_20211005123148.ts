import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children: React.ReactElement): RenderResult => {
  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider> );
};
