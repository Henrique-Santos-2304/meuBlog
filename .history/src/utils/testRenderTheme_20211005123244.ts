import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children) => {
  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider> );
};
