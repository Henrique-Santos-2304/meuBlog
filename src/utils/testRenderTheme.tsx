import { ThemeProvider } from "styled-components";
import { ChakraProvider } from "@chakra-ui/react";

import { render, RenderResult } from "@testing-library/react";
import theme from "styles/theme";
export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(
    <ThemeProvider theme={theme}>
      <ChakraProvider> {children}</ChakraProvider>
    </ThemeProvider>
  );
};
