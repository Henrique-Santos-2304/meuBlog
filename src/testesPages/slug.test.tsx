import { renderTheme } from "utils/testRenderTheme";
import Slug from "pages/projects/[slug]";

describe("<BoxMain />", () => {
  it("should render ", () => {
    const { container } = renderTheme(<Slug />);
    expect(container).toMatchSnapshot();
  });
});
