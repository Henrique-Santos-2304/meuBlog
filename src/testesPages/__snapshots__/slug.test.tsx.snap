// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoxMain /> should render  1`] = `
<div>
   
  <div
    class="chakra-container css-p2xluk"
  >
    <div
      class="chakra-container css-nm5t63"
    >
      Teste
    </div>
  </div>
</div>
`;
